github/filecoin-project.yml

```diff
@@ -2517,16 +2444,7 @@ repositories:
     collaborators:
       admin:
         - anorth
-        - jennijuju
-        - Kubuxu
-        - masih
```

Comment:
At least one active maintainer of `go-f3` either @Stebalien, @kubuxu or @masih should be admin.

Reply:
I know the GitHub diff doesn't make this clear, but what happened here is that lotus-maintainers has admin and I removed individuals (besides anorth since he isn't in lotus-maintainers).  As a result I think we're good.